open-backport-pull-request/src/run.ts

```diff
@@ -22,37 +20,30 @@ type Outputs = {
 }
 
 export const run = async (inputs: Inputs): Promise<Outputs | undefined> => {
-  const octokit = getOctokit(inputs.githubToken, octokitPluginRetry.retry)
-  const { baseBranch, headBranch } = inputs
-
-  if (await hasDiff({ headBranch, baseBranch, octokit, context })) {
-    return await openPullRequest(
```

Comment:
Moved to line 37 https://github.com/quipper/monorepo-deploy-actions/pull/1435/files#diff-f2944a3a81132688fe8339ed478459a84154c64c8c2753c2f96f0a74a067df83R37